administrator/language/en-GB/plg_system_schemaorg.ini

```
@@ -0,0 +1,16 @@
+PLG_SYSTEM_SCHEMAORG="Schema.org System Plugin"

```

Please change this to be consistent with the naming of all the other system plugins


```suggestion
PLG_SYSTEM_SCHEMAORG="System - Schema.org"
```